Controller/CRUDController.php

```diff
@@ -296,7 +295,6 @@ public function batchActionDelete(ProxyQueryInterface $query)
      *
      * @return Response|RedirectResponse
      *
-     * @throws NotFoundHttpException If the object does not exist
```

Comment:
No, it's thrown in this method. The exception is generated in the parent method. Seems that phpstrom needs some improvements :)